src/how-to-start-contributing.md

```diff
@@ -0,0 +1,134 @@
+[analyzer-issues]: https://github.com/rust-lang/rust-analyzer/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22
+[miri-repo]: https://github.com/rust-lang/miri
+[miri-guide]: https://github.com/rust-lang/miri/blob/master/CONTRIBUTING.md
+[miri-issues]: https://github.com/rust-lang/miri/issues?q=is%3Aissue%20state%3Aopen%20label%3AE-good-first-issue
+[rustup-repo]: https://github.com/rust-lang/rustup
+[rustup-guide]: https://rust-lang.github.io/rustup/dev-guide
+[rustup-issues]: https://github.com/rust-lang/rustup/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22
+[crates-io-repo]: https://github.com/rust-lang/crates.io
+[crates-io-guide]: https://github.com/rust-lang/crates.io/blob/main/docs/CONTRIBUTING.md
+[crates-io-issues]: https://github.com/rust-lang/crates.io/issues
+[bors-repo]: https://github.com/rust-lang/bors
+[bors-guide]: https://github.com/rust-lang/bors/blob/main/docs/development.md
+[bors-issues]: https://github.com/rust-lang/bors/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22
+[rustc-perf-repo]: https://github.com/rust-lang/rustc-perf
+[rustc-perf-issues]: https://github.com/rust-lang/rustc-perf/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22
+[triagebot-repo]: https://github.com/rust-lang/triagebot
+[triagebot-issues]: https://github.com/rust-lang/triagebot/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22
+[playground-repo]: https://github.com/rust-lang/rust-playground
+[playground-issues]: https://github.com/rust-lang/rust-playground/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22
+[rustlings-repo]: https://github.com/rust-lang/rustlings
+[rustlings-guide]: https://github.com/rust-lang/rustlings/blob/main/CONTRIBUTING.md
+[rustlings-issues]: https://github.com/rust-lang/rustlings/issues
+[mdbook-repo]: https://github.com/rust-lang/mdBook
+[mdbook-guide]: https://github.com/rust-lang/mdBook/blob/master/CONTRIBUTING.md
+[mdbook-issues]: https://github.com/rust-lang/mdBook/labels/E-Help-wanted
+
+If you want to get inspired, check all the [rust-lang][rust-lang-repos] repositories!
+
+[rust-lang-repos]: https://github.com/orgs/rust-lang/repositories?type=all&q=sort%3Astars
+
+### Different kinds of contributions
+
+There are various ways in which you can contribute to Rust projects:
+
+- Writing code is the most obvious one. However, it does not have to be only in Rust! Even though
+  most of our projects are of course written in Rust, we also use other technologies. For example,
+  you can help improve our GitHub CI workflows, automation Python scripts or contribute to web
+  frontends with HTML/CSS/JS (e.g. [Rustdoc][rustdoc-ui] or [Benchmark suite website][rustc-perf-ui]).
+  Play to your strengths!
+- Improving documentation is possibly one of the easiest starting points for contributions.
+  Did you find a typo, something that was unclear, or do you miss some useful information on this
```

Comment:
```suggestion
  Did you find a typo in user-facing material, something that was unclear, or do you miss some useful information on this
```

Reply:
At the end of the sentence I explicitly stated that it's about documentation, but I guess someone could interpret that as also e.g. "comments in rustc" and I know that these kinds of PRs are not ideal :D